bot/exts/easter/earth_photos.py

```diff
@@ -0,0 +1,46 @@
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+from bot.constants import Tokens
+
+UnClient_id = Tokens.unsplash_key
+
+
+class EarthPhotos(commands.Cog):
+    """This cog contains the command for earth photos."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["earth"])
+    async def earth_photos(self, ctx: commands.Context) -> None:
+        """Returns a random photo of earth, sourced from Unsplash."""
+        async with ctx.typing():
+            async with self.bot.http_session.get(
+                    'https://api.unsplash.com/photos/random',
+                    params={"query": "earth", "client_id": UnClient_id}) as r:
+                jsondata = await r.json()
+                linksdata = jsondata.get("urls")
+                embedlink = linksdata.get("regular")
+                downloadlinksdata = jsondata.get("links")
+                userdata = jsondata.get("user")
+                username = userdata.get("name")
+                userlinks = userdata.get("links")
```

Comment:
Can you remove the redundant variables here? Like the `userlinks `.

Reply:
I have to keep them for getting specific data, unless there's a better way, I don't work with web requests much

Reply:
`userlinks ` is being used anywhere and even a few others so I don’t see a need of them.

Reply:
userlinks is used to get profile link for embed, downloadlinksdata for the download links, linksdata for the embed links, embedlink is self-explanitory, userdata is also self explanitory, username is for crediting the person who made the image as of [here](https://help.unsplash.com/en/articles/2511245-unsplash-api-guidelines), name crediting is needed for production app, profile is used for a link to the profile